x-team/xp-ui
# Add JobApplicationCard component

**Release Type:** *Non-Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3373

## Description

Add JobApplicationCard component.


## Checklist


## Steps to Test or Reproduce

Check out the storybook site and look at `JobApplicationCard` component stories. Compare to mockup images on the ticket
[XP-3373](https://x-team-internal.atlassian.net/browse/XP-3373)
and on design
https://www.figma.com/file/ilfpDrEx9nKf7BXEQffGqu/XP-Jobs

## Screenshots

<img width="713" alt="Screen Shot 2019-10-21 at 8 41 56 PM" src="https://user-images.githubusercontent.com/15269623/67257979-58b8c100-f443-11e9-8e82-ae9327f5a4f7.png">
<img width="721" alt="Screen Shot 2019-10-21 at 8 42 05 PM" src="https://user-images.githubusercontent.com/15269623/67257985-5d7d7500-f443-11e9-9f80-9cf1844a2216.png">
